src/java.base/share/classes/java/time/MonthDay.java

```diff
@@ -675,7 +675,11 @@ public LocalDate atYear(int year) {
      * It is "consistent with equals", as defined by {@link Comparable}.
      *
      * @param other  the other month-day to compare to, not null
-     * @return the comparator value, negative if less, positive if greater
+     * @return the comparator value is less than zero if the {@code other} is before,
```

Comment:
Using before/after here could be confusing, as January could be considered to be before or after July (since the year is not defined).

Reply:
The `isBefore` and `isAfter` methods use that terminology and Month is an enum with defined order January thru December and use the `compareTo` method to compute before/after.